rasa/nlu/utils/__init__.py

```
@@ -315,6 +315,41 @@ def read_endpoints(endpoint_file: Text) -> "AvailableEndpoints":
     return AvailableEndpoints(model, data)
 
 
+def validate_pipeline_yaml(yaml):
+    """Validate NLU pipeline yaml."""
+    from pykwalify.core import Core
+    import pkg_resources
+    from pykwalify.errors import SchemaError
+    from ruamel.yaml import YAMLError
+    from rasa.nlu.model import InvalidModelError
+
+    log = logging.getLogger("pykwalify")
+    log.setLevel(logging.WARN)
+
+    try:
+        schema_file = pkg_resources.resource_filename(__name__, "schemas/nlu_model.yml")
+        source_data = rasa.utils.io.read_yaml(yaml)
+    except YAMLError:
+        raise InvalidModelError(
```

Can you raise a `InvalidConfigError` instead?